src/main/java/io/supertokens/config/CoreConfig.java

```diff
@@ -275,15 +275,16 @@ public class CoreConfig {
                     " address.")
     private String ip_deny_regex = null;
 
-    @ConfigYamlOnly
+    @NotConflictingInApp
     @JsonProperty
     @HideFromDashboard
     @ConfigDescription(
             "If specified, the core uses this URL to connect to the OAuth provider public service.")
     private String oauth_provider_public_service_url = null;
 
-    @ConfigYamlOnly
+    @NotConflictingInApp
     @JsonProperty
+    @HideFromDashboard
     @ConfigDescription(
```

Comment:
Need to modify the normalise function in this file to do lower case for the urls?

Reply:
According to google and w3c, urls should be case sensitive (https://www.w3.org/TR/WD-html40-970708/htmlweb.html). For this reason I would rather not lowecase them